api/lib/use-cases/FetchRecords.js

```diff
@@ -0,0 +1,32 @@
+const mergeResponses = require('../../lib/MergeResponses');
```

Comment:
I think mergeResponses should also be injected so it can be stubbed, unless you think it should be part of FetchRecords? If it's kept separate we should't be testing it's implementation in unit tests (as we are below)